Windows/resource.h

```diff
@@ -319,6 +318,11 @@
 #define ID_OPTIONS_BUFNEARESTFILTER      40153
 #define ID_OPTIONS_DIRECT3D9             40154
 #define ID_OPTIONS_OPENGL                40155
+#define ID_EMULATION_ROTATION_H          40156
+#define ID_EMULATION_ROTATION_V          40157
+#define ID_EMULATION_ROTATION_H_R        40158
+#define ID_EMULATION_ROTATION_V_R        40159
+#define ID_OPTIONS_DISPLAY_LAYOUT        40160
```

Comment:
Also not a big thing, but if you're visiting this code, maybe also change `_APS_NEXT_COMMAND_VALUE` to `40161` to keep the visual editor happy.

-[Unknown]